templates/etcd/etcd.py

```diff
@@ -109,4 +109,7 @@ def update_cluster(self, cluster):
             self._etcd_sal.stop()
             self._etcd_sal.start()
         except StateCheckError:
-            pass
\ No newline at end of file
+            pass
+
+    def enable_auth(self):
```

Comment:
make it a private method cause this is not a action that an end user should use
```suggestion
    def _enable_auth(self):
```